packages/devtools/lib/src/memory/memory.dart

```diff
@@ -96,13 +105,35 @@ class MemoryScreen extends Screen with SetStateMixin {
 
     final CoreElement screenDiv = div(c: 'custom-scrollbar')..layoutVertical();
 
+    html.window.onKeyDown.listen((html.KeyboardEvent e) {
+      if (e.ctrlKey) {
+        switch (e.key) {
+          case 'f': // CTRL + f
+            _search();
+            e.preventDefault();
+            break;
+        }
+      }
+    });
+
     resumeButton = PButton.icon('Resume', FlutterIcons.resume_white_disabled_2x)
       ..primary()
       ..small()
       ..disabled = true;
 
     pauseButton = PButton.icon('Pause', FlutterIcons.pause_black_2x)..small();
 
+    heapPopupList = PopupListView<String>();
+
+    vmSearchField = CoreElement('input', classes: 'search-text')
+      ..setAttribute('type', 'text')
+      ..setAttribute('placeholder', 'filter')
+      ..id = 'popup_search_memory';
```

Comment:
do you have to use an id? in general use a css class is a bit better as you can have more than one element with the same class in the dom.

Reply:
Used that for some debugging and forgot about it - its never used. Gone now.